frontend/app/src/main/java/dk/scheduling/schedulingfrontend/components/StandardDropDownMenu.kt

```diff
@@ -46,7 +47,7 @@ fun StandardDropDownMenu(
                 modifier
                     .fillMaxWidth()
                     .menuAnchor(),
-            value = selectedItem,
+            value = options[selectedItem]!!,
```

Comment:
Sus

Reply:
It's fine 
![image](https://github.com/cs-24-sw-8-04/scheduling/assets/72788863/c39e010b-9ceb-4e97-9125-2e10318e94a0)